src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -52,11 +52,10 @@
 <ng-template #statusColor
              let-value="value">
   <span *ngFor="let state of value; last as last">
-    <span [class.text-success]="'up' === state || 'in' === state"
-          [class.text-warning]="'down' === state || 'out' === state">
+    <span [class.success-text]="'up' === state || 'in' === state"
```

Comment:
Please try to use bootstrap classes whenever possible.
I think we should keep using `text-success` here and use `text-danger` instead of `critical-text`

Reply:
I can do that, but although possible, the color is not as bright as before and hence it's not as obvious as before that the OSD is down. Personally, I'd prefer the brighter red.

![osd_down_text_danger](https://user-images.githubusercontent.com/9606095/44771447-44dd3c00-ab6b-11e8-9c36-e12ecdb6d046.png)

Reply:
The not so bright red is OK for me as the heart icon is also displayed, this way it's easy to spot if something in the cluster isn't working as expected by the dashboard. Also you can filter the OSD list by `status:down`.

Reply:
As per our discussion in the standup, the states will be converted to labels.